docs/README.md

```diff
@@ -284,6 +284,7 @@ Check out the [**Product Summary documentation**](https://vtex.io/docs/component
 
 | Prop name | Type                      | Description                                                                                       | Default value |
 | --------- | ------------------------- | ------------------------------------------------------------------------------------------------- | ------------- |
+| `categoryFiltersMode`  | `Enum` | Whether the Category Filters should use href (`href`) or not (`default`). By default the filter uses divs with role="link" you may change to href to improve store Link Building. | `default`  |
```

Comment:
I didn't understand the meaning behind _You may change to href to improve store Link Building._ Could you please clarify this prop description for me? :)

Reply:
@MatheusR42 is not around now, but he told me that this can improve the google shop index, I don't know if that is clear to you

Reply:
@guerreirobeatriz , Link Building is a technique to improve website SEO. In short when many pages link to a page "X" the page "X" will improve rank score. With the "href" feature enabled we will make every category page to link  to child categories so all categories will improve its page rank.